core/sr-primitives/src/lib.rs

```diff
@@ -374,6 +537,127 @@ impl From<codec::Compact<Perbill>> for Perbill {
 	}
 }
 
+/// Representation of a sign bit.
+///
+/// Note that the ordering matters for proper `Ord` derive.
+#[cfg_attr(feature = "std", derive(Debug))]
+#[derive(PartialEq, Eq, PartialOrd, Ord)]
+pub enum Sign {
+	/// Negative
+	N,
+	/// Positive
+	P,
+}
+
+/// A fixed point number in the range of `[-(u64::max+1), u64::max+1]`. Composed of one `u64`
+///  _natural_ part and a `Perbill` _decimal_ part.
```

Comment:
yes I'm mixed with decimal because it kind of means x/10 no ? but at the same time the mention of perbill makes it clear

Reply:
agree the name might be misleading. at least with `n` I tried to deliver the message that it is a natural number which should clarify it.

Reply:
maybe _fractional_ is a good word 🤔

https://en.wikipedia.org/wiki/Fraction_(mathematics)#Mixed_numbers